### PR TITLE
Sentence case all nav files

### PR DESCRIPTION
**Pull request by bradleycamacho**
Sentence cases all `meta.json` files (nav files) in accordance with our style guide.

Reviewer, please take an extra moment to make sure I correctly capitalized the files. There were some instances where I had to make a judgement call :)

### CODE REVIEW

File: pages/builders/app-developers/tutorials/_meta.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+  "first-contract": "Deploying your first contract on OP Mainnet",
+  "cross-dom-solidity": "Communicating between chains in Solidity",
+  "cross-dom-bridge-eth": "Bridging ETH with the Optimism SDK",
+  "cross-dom-bridge-erc20": "Bridging ERC-20 tokens with the Optimism SDK",
+  "standard-bridge-custom-token": "Bridging your custom ERC-20 token to OP Mainnet",
+  "standard-bridge-standard-token": "Bridging your standard ERC-20 token to OP Mainnet",
+  "sdk-view-txns": "Viewing deposits and withdrawals by address",
+  "sdk-trace-txns": "Tracing deposits and withdrawals",
+  "sdk-estimate-costs": "Estimating transaction costs",
+  "send-tx-from-eth": "Triggering OP Mainnet transactions From Ethereum"
```
Comment (sbvegan) [inline pages/builders/app-developers/tutorials/_meta.json:11]:
```suggestion
  "send-tx-from-eth": "Triggering OP Mainnet transactions from Ethereum"
```

File: pages/builders/chain-operators/tutorials/_meta.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-    "create-l2-rollup": "Creating Your Own L2 Rollup Testnet",
+    "create-l2-rollup": "Creating your own L2 Rollup Testnet",
```
Comment (sbvegan) [inline pages/builders/chain-operators/tutorials/_meta.json:2]:
```suggestion
    "create-l2-rollup": "Creating your own rollup testnet",
```

File: pages/chain/_meta.json [hunk 3]
```diff
@@ -1,9 +1,9 @@
 {
-  "getting-started": "Getting Started: OP Mainnet",
-  "differences": "Differences Between Ethereum and OP Mainnet",
+  "getting-started": "Getting started: OP Mainnet",
+  "differences": "Differences detween Ethereum and OP Mainnet",
```
Comment (sbvegan) [inline pages/chain/_meta.json:3]:
```suggestion
  "differences": "Differences between Ethereum and OP Mainnet",
```

**Review by sbvegan — APPROVED** [review]
a few small fixes, but once those are updated, you're free to merge

**Comment by bradleycamacho** [comment]
@sbvegan sorry totally missed your review, thanks for fixing those!

**Pull request merged**